src/System.Device.Gpio/Interop/Unix/libgpiod/V2/GpiodException.cs

```diff
@@ -3,17 +3,29 @@
 
 namespace System.Device.Gpio.Libgpiod;
 
-internal class GpiodException : Exception
+/// <summary>
+/// Exception in the context of calling libgpiod
+/// </summary>
+public class GpiodException : Exception
```

Comment:
GPIO communication errors so far throw `IOException`, so this should probably derive from that. 

Additionally, the namespace is wrong, should be the same as the public LibGpiodDriver.

Reply:
changed namespace/file location and let it derive from IOException